website/src/stories/input.stories.tsx

```
@@ -0,0 +1,60 @@
+import { Input } from '@/components/input';
+
+import type { Meta, StoryObj } from '@storybook/nextjs-vite';
+
+const meta = {
+	title: 'Components/Input',
+	tags: ['autodocs'],
+	component: Input,
+	args: {
+		placeholder: 'Enter text',
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)